WalletWasabi/Services/WalletService.cs

```diff
@@ -1103,8 +1118,17 @@ public Operation(Script script, Money amount, string label)
 			Money feePerBytes = null;
 			using (var client = new WasabiClient(Synchronizer.WasabiClient.TorClient.DestinationUriAction, Synchronizer.WasabiClient.TorClient.TorSocks5EndPoint))
 			{
-				Money feeRate = Synchronizer.GetFeeRate(feeTarget);
-				Money sanityCheckedFeeRate = Math.Max(feeRate, 2); // Use the sanity check that under 2 satoshi per bytes should not be displayed. To correct possible rounding errors.
+				Money fr;
+				if (feeTarget.HasValue)
+				{
+					fr = Synchronizer.GetFeeRate(feeTarget.Value);
+				}
+				else
+				{
+					fr = feeRate;
+				}
```

Comment:
Conditional expression can be used.
`Money fr = feeTarget.HasValue ? Synchronizer.GetFeeRate(feeTarget.Value) : feeRate;`